app/forms/candidate_interface/reference/request_referee_email_address_form.rb

```diff
@@ -0,0 +1,16 @@
+module CandidateInterface
+  class Reference::RequestRefereeEmailAddressForm < Reference::RefereeEmailAddressForm
+    def email_address_unique
+      other_references.each do |other_reference|
+        next if other_reference.cancelled? || other_reference.cancelled_at_end_of_cycle?
+
+        errors.add(:email_address, :"duplicate.#{other_reference.feedback_status}") if other_reference.email_address.downcase == email_address.downcase
+      end
```

Comment:
i have a marginal preference for
```
      if other_references
          .reject { |ref| ref.cancelled? || ref.cancelled_at_end_of_cycle? }
          .any? { |ref| ref.email_address.downcase == email_address.downcase }
        errors.add(:email_address, :"duplicate.#{other_reference.feedback_status}")
      end
```

Reply:
Good idea, I changed slightly the code as I need the other_reference object. Let me know your thoughts